README_GAUDI.md

```diff
@@ -11,7 +11,7 @@ Please follow the instructions provided in the [Gaudi Installation Guide](https:
 - OS: Ubuntu 22.04 LTS
 - Python: 3.10
 - Intel Gaudi accelerator
-- Intel Gaudi software version 1.18.0
+- Intel Gaudi software version 1.19.0
 
 ## Quick start using Dockerfile
 ```
```

Comment:
It needs more explanation.

Reply:
Will you add it? @piotrbocian

Reply:
Or explain what you have in mind so someone else can do it?

Reply:
Other vendors seem to have it in the similar way as here:
https://docs.vllm.ai/en/latest/getting_started/openvino-installation.html#quick-start-using-dockerfile
https://docs.vllm.ai/en/latest/getting_started/cpu-installation.html#quick-start-using-dockerfile
https://docs.vllm.ai/en/latest/getting_started/arm-installation.html#quick-start-with-dockerfile
https://docs.vllm.ai/en/latest/getting_started/xpu-installation.html#quick-start-using-dockerfile

Reply:
Please see how document is structured:

1. Quick start using Dockerfile
2. Build from source
 2.1 Environment verification
 2.2 Run Docker Image
 2.3 Build and Install vLLM

Questions:
- is (1.) full alternative to (2.)? If so, I would add one liner as
- "You can quickly set up vLLM using latest Intel Gaudi docker and vllm verson <TBD>"
- is (2.1 Env verification) common to (1.) and (2.)?